contributing.md

```diff
@@ -258,6 +258,10 @@ Example: `Fix typos in Foo scaladoc`
         1. **Component**
         1. **Affects Version**. For Bugs, assign at least one version that is known to exhibit the 
         problem or need the change
+        1. **Label**. Not widely used, except for the following:
+             - `correctness`: a data loss or correctness issue
```

Comment:
I feel perhaps we should use a different label for data loss?

Reply:
Data loss is part of correctness issues, right?